### ISSUE TITLE
Upload your Files here otherwise we have to report you because it could be possible you advertise a virus. Thanks!

### ISSUE THREAD

**Issue by einfacherNico** [Line2x2/LunacyDM#1]

**Comment by E1r0y** [comment]
Its a virus. I had to reinstall my windows after runnig the exe file

**Comment by E1r0y** [comment]
![image](https://github.com/Line2x2/LunacyDM/assets/54291200/16ed43f8-3537-4b60-b356-8064ac5a800c)
![image](https://github.com/Line2x2/LunacyDM/assets/54291200/11f6081e-56f9-49f0-925c-aad25a76aeb4)
![image](https://github.com/Line2x2/LunacyDM/assets/54291200/d54dd9de-ceef-40a8-9192-5a97cf2bede5)

**Comment by E1r0y** [comment]
I send report

**Comment by einfacherNico** [comment]
Yes, it was kinda obvious because hes only showing it on Telegram. Its crazy how people have no feelings and can be shameless like this.
